_data/52.yml

```diff
@@ -0,0 +1,90 @@
+name: Leap Micro
+version: 5.2
+bg-color: leapmicro
+fg-color: white
+logo: logos/leapmicro.svg
+choosing-media: true
+downloads:
+- name: installation_images
+  display: all
+  arches:
+  - name: x86_64
+    types:
+    - name: offline_image
+      desc: leapmicro_desc
+      primary_link: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-x86_64-Media.iso
+      links:
+      - name: metalink
+        url: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-x86_64-Media.iso.meta4
+      - name: pick_mirror
+        url: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-x86_64-Media.iso?mirrorlist
+      - name: checksum
+        url: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-x86_64-Media.iso.sha256
+  - name: aarch64
+    types:
+    - name: offline_image
+      desc: leapmicro_desc
+      primary_link: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-aarch64-Media.iso
+      links:
+      - name: metalink
+        url: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-aarch64-Media.iso.meta4
+      - name: pick_mirror
+        url: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-aarch64-Media.iso?mirrorlist
+      - name: checksum
+        url: /distribution/leap-micro/5.2/product/iso/openSUSE-Leap-Micro-5.2-DVD-aarch64-Media.iso.sha256
+- name: vm_images
+  arches:
+  - name: x86_64
+    types:
+    - name: default
```

Comment:
Keep in mind that those keys are then used in `_data/locales/en.yml` to show some text, and are also used as icon names displayed next to the download. Feel free to symlink the appropriate icons in `_includes/icons`, to make that work, but please also add proper translations to the aforementioned file

Reply:
Yeah, you need to define the keys and fill out their values.